ci/prtest.config

```diff
@@ -1,6 +1,6 @@
 RUN_CUGRAPH_LIBCUGRAPH_TESTS=OFF
 RUN_CUGRAPH_PYTHON_TESTS=OFF
 
-RUN_CUML_LIBCUML_TESTS=OFF
-RUN_CUML_PRIMS_TESTS=OFF
-RUN_CUML_PYTHON_TESTS=OFF
+RUN_CUML_LIBCUML_TESTS=ON
+RUN_CUML_PRIMS_TESTS=ON
+RUN_CUML_PYTHON_TESTS=ON
```

Comment:
The `ops` team is tagged as a `codeowner` for this file, but I'm not sure what the preferred value is for these variables. I'll defer to @dantegd for this change since it seems he's touched this file last.

Reply:
I don't believe this file is even being used currently but we should probably revert these changes so we're not introducing unneeded changes into this PR